content/docs/ai/ai-intro.md

```diff
@@ -7,18 +7,129 @@ updatedOn: '2023-10-07T10:43:33.361Z'
 
 Vector databases enable efficient storage and retrieval of vector data, which is an essential component in building AI applications that leverage Large Language Models (LLMs) such as OpenAI.
 
-Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
+Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open-source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
 
-To get started, see [The pgvector extension](/docs/extensions/pgvector).
+<CTA title="Neon AI Starter Kit" description="Get a headstart on your AI app with the Neon AI Starter Kit. Tools, starter apps, and step-by-step instructions to get you up and running in minutes." buttonText="Sign Up" buttonUrl="https://console.neon.tech/signup" />
 
-## Example applications
+The _Neon AI Starter Kit_ includes:
 
-Check out the following AI application examples built with Neon.
+- Neon Postgres with the latest version of `pgvector` for storing vector embeddings
+- The `neon_vecs` Python and JavaScript SDKs for managing and querying vector embeddings in Postgres with the `pgvector` extension
+- The `neon_ai` utility for generating vector embeddings with your chosen LLM and storing them in Postgres
+- Starter apps to get you up and running in minutes:
+  - AI chatbot starter app
+  - RAG starter app
+  - Semantic search starter app
+  - Hybrid search starter app
+  - Image search starter app
+  - Chat with PDF starter app
+- The Neon AI scaling guide
```

Comment:
can we expand on this in here - helps users like me know what to expect if I click on it.